### ISSUE TITLE
increase indicator contrast on light gtk3 themes

### ISSUE THREAD

**Issue by hrdwrrsk2** [nsz32/docklike-plugin#101]
the indicator color (when window is unfocused/minimized) contrast in light themes are kinda hard to see for me. it should be customizable like active indicator color or share indicator color between active/minimized apps (like dash-to-panel gnome extension did), but i would like to make the indicator color and active/focused background customizable with Gtk.CSS (aka theme integration)

![Screenshot_2021-06-03_08-45-52](https://user-images.githubusercontent.com/70018316/120573663-26a57880-c448-11eb-978e-6e66f72b3e86.png)

**Comment by davekeogh** [comment]
Hi, I'm not too familiar with Gtk3 CSS. Am I correct in thinking that if all the widgets have their own unique style classes and/or ids, they can be themed in this way? Then it just needs to be documented somewhere.

**Comment by hrdwrrsk2** [comment]
> Hi, I'm not too familiar with Gtk3 CSS. Am I correct in thinking that if all the widgets have their own unique style classes and/or ids, they can be themed in this way? Then it just needs to be documented somewhere.

yeah, i mean the indicator and hover/active background should be themeable using css, allowing to use transparent (rgba) bg color or some gradient(s). for (unfocused) indicator i think it should be adapt with background lightness from the theme or from custom color setting (if the panel bg is white (`#fff`), the unfocused indicator will darker or vice versa)

**Comment by davekeogh** [comment]
I took a look at this a bit today, but it's more complex than I had originally thought. The code changes some widgets styles internally and results in some visual bugs when overriding the CSS. None of this code was written by myself so it is a bit difficult to understand. I'll try to look into this some more at a later date.

I added a class name for the plugin itself here:
https://github.com/davekeogh/xfce4-docklike-plugin/commit/52ff212f7e91d46982416cd7cbf8ec2616e8dc9b

So you can select the buttons like this with CSS:
`#docklike-plugin .group { background-color: lightblue; }`

You will probably notice some breakage pretty fast when you start interacting with the buttons though.
